pidstat.h

```diff
@@ -15,9 +18,9 @@
 
 #define NR_PID_PREALLOC	100
 
-#define MAX_COMM_LEN	128
-#define MAX_CMDLINE_LEN	128
-#define MAX_USER_LEN	32
+#define MAX_COMM_LEN    _POSIX_ARG_MAX
```

Comment:
Do all platforms define `_POSIX_ARG_MAX` and the others?  Seems like this might need platform protection.